File: services/Games.API/Models/Game.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Games.API.Models
{
    public class Game
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public Guid Cover { get; set; }
        public Guid Thumbnail { get; set; }
        public DateTime DateCreation { get; set; }
        public DateTime DateModification { get; set; }
    }
}

File: apps/VueApp/src/models/Game/game.ts
export class GameDTO {
  public id = 0;
  public title = '';
  public cover = '';
  public thumbnail = '';
  public is_favorited_by_user = false;
}

export class Game extends GameDTO {
  constructor(data?: GameDTO) {
    super();
    Object.assign(this, data || new GameDTO());
  }
}
